src/classes/AccountDeDuper.cls

```diff
@@ -1,9 +1,16 @@
-public with sharing class AccountDeDuper {
+public with sharing class Account_De_Duper {
+  // new stuff
+  // more stuff
 
   // Constructor.
 Public AccountDeDuper () {
 
 }
+public void badStuff() {}
+public void badStuff2() {}
+public void BADSTUFF3() {}
+public void BADSTUFF__4() {}
```

Comment:
* This method is empty, I suggest to remove it.
* Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.